QueryAI/JiraV2.py

```diff
@@ -0,0 +1,697 @@
+from core.queryaiCommonServerPython import *
+import core.queryaiDemistomock as demisto
+
+''' IMPORTS '''
+import json
+import requests
+from requests_oauthlib import OAuth1
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
```

Comment:
Do we need this? Won't we have certificates installed?

Reply:
This was there in the original file so I didn't modify, you can change if required.